maps_dashboards/public/components/layer_config/document_layer_style.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect } from 'react';
+import { EuiFormRow, EuiColorPicker, useColorPickerState } from '@elastic/eui';
+import { DocumentLayerSpecification } from '../../model/mapLayerType';
+
+interface Props {
+  selectedLayerConfig: DocumentLayerSpecification;
+  setSelectedLayerConfig: Function;
+}
+
+export const DocumentLayerStyle = ({ setSelectedLayerConfig, selectedLayerConfig }: Props) => {
+  const [color, setColor, errors] = useColorPickerState(
+    selectedLayerConfig?.style?.fillColor || '#D36086'
```

Comment:
Shall we generate random color here?

Reply:
Yes, changed to get random color when generating initial layer config.